internal/utils/helper.go

```diff
@@ -34,25 +35,32 @@ var (
 	locker         sync.Mutex
 )
 
-func MessageClientID() string {
+func init() {
+	buf := new(bytes.Buffer)
+
+	ip, err := clientIP4()
+	if err != nil {
+		ip = fakeIP()
+	}
+	buf.Write(ip)
+	binary.Write(buf, binary.BigEndian, Pid())
```

Comment:
use `_ = binary.Write(buf, binary.BigEndian, Pid())` to ignore the error if must not occur

Reply:
done